lms/djangoapps/ora_staff_grader/ora_api.py

```diff
@@ -33,6 +34,36 @@ def get_submissions(request, usage_id):
     return json.loads(response.content)
 
 
+def get_assessments_given(request, usage_id, submission_uuid):
+    """
+    Get a list of assessments given from the ORA's 'list_assessments_given' XBlock.json_handler
+    """
+    handler_name = "list_assessments_given"
+    data = {"item_id": usage_id, "submission_uuid": submission_uuid}
```

Comment:
```suggestion
    data = {
        "item_id": usage_id,
        "submission_uuid": submission_uuid,
   }
```
This reads better. Try using this kind of style in the rest of the implementation if you can